Resources/doc/cookbook/recipe_lock_protection.rst

```diff
@@ -0,0 +1,31 @@
+Lock protection
+==========================
+
+Lock protection will prevent data corruption when multiple users edit an object at the same time.
+
+Example
+^^^^^^^
+
+1) Alice starts to edit the object
+2) Bob starts to edit the object
+3) Alice submits the form
+4) Bob submits the form
+
+In this case, a message will tell Bob that someone else has edited the object,
+and that he must reload the page and apply the changes again.
+
+Enable lock protection
+----------------------
+
+By default, lock protection is disabled. You can enable it in your ``sonata_admin`` configuration :
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        sonata_admin:
+            options:
+                lock_protection: true
+
+.. note::
+    Lock protection must be supported by the corresponding model manager.
```

Comment:
@OskarStark The ``SonataDoctrineORMAdminBundle`` is not installed/required when you install the ``SonataAdminBundle`` (it's not the "default" admin doctrine bundle).

However, we may add a note saying that currently only the ``SonataDoctrineORMAdminBundle`` supports it.